jscatter/widget.py

```diff
@@ -185,6 +196,20 @@ class JupyterScatter(anywidget.AnyWidget):
     # For synchronyzing view changes across scatter plot instances
     view_sync = Unicode(None, allow_none=True).tag(sync=True)
 
+    def __init__(self, data, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+        self.data = data
+        self.on_msg(self._handle_custom_msg)
+
+    def _handle_custom_msg(self, data: dict, buffers):
+        if data["type"] == TOOLTIP_EVENT_TYPE and isinstance(self.data, pd.DataFrame):
```

Comment:
and this message is debounced on the frontend, right?

Reply:
Yeah correct